js/models/ShippingAddress.js

```diff
@@ -2,7 +2,7 @@ import BaseModel from './BaseModel';
 import app from '../app';
 import is from 'is_js';
 
-export default class extends BaseModel {
+export default class ShippingAddress extends BaseModel {
```

Comment:
In either case it's a problematic approach because if we ever obfuscated the code during the build process, the class name would change.